libs/platform/i18n/src/lib/locale/default-locale.service.ts

```diff
@@ -57,21 +57,34 @@ export class DefaultLocaleService implements LocaleService {
     this.setActive$.next(value);
   }
 
-  formatDate(stamp: string | number, showTime = false): Observable<string> {
+  formatDate(
+    stamp: string | number | Date,
+    showTime = false,
+    showDate = false
+  ): Observable<string> {
```

Comment:
Can we split this method into multiple once to remove those boolean flags as it's really unclear what that means in other places when it's being used.
We can have at least 2 separate methods `formatDate` and `formatTime`.
If needed you may create the third one `formatDateTime` which will render both date and time but if it's not used anywhere no need for it.